daprdocs/content/en/reference/components-reference/supported-pubsub/setup-jetstream.md

```diff
@@ -146,6 +143,31 @@ It is essential to create a NATS JetStream for a specific subject. For example,
 nats -s localhost:4222 stream add myStream --subjects mySubject
 ```
 
+## Example: Competing consumers pattern
+
+Let's say you'd like each message to be processed by only one application or pod with the same app-id. Typically, the `consumerID` metadata spec helps you define competing consumers. However, `consumerID` is not supported in NATS JetStream.
+
+The following example demonstrates using a competing consumer pattern with dynamic values. 
+
+```yml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: pubsub
+spec:
+  type: pubsub.jetstream
+  version: v1
+  metadata:
+  - name: name
+    value: "my-conn-name"
+  - name: streamName
+    value: "my-stream"
+  - name: durableName 
+    value: "my-durable"
```

Comment:
Maybe giving it a more meaningful name would be helpful?
Same for `queueGroupName`.

I'm not sure how you do this with other examples, but my intuition is that if this stands out more, it would be even easier to understsand.

```suggestion
    value: "events_group"
```